calico_node/tests/startup_test.py

```diff
@@ -1,153 +0,0 @@
-# Copyright (c) 2015-2016 Tigera, Inc. All rights reserved.
```

Comment:
We should probably have the test infrastructure in place to reinstate these tests as part of this PR, even if we don't get full coverage.

That way we can at least easily add tests in the future.